core/expressionParser/expression.js

```diff
@@ -17,6 +17,12 @@ export const modes = {
   sql: 'sql',
 }
 
+export const nodeProperties = {
+  length: 'length',
+}
```

Comment:
these properties must be taken from native javascript types.

Reply:
yes, but we should know somehow, what are the "allowed" properties, the ones we support, to have control on the expressions written by the user. I'll move them to a separate file.